backend/docs/developers/setup-local-env.rst

```diff
@@ -43,19 +44,36 @@ You need the following libraries and/or programs:
        npm run build
 
 #. Run the migrations with ``src/manage.py migrate``
+
+.. note::
+
+       If you get an error about the ``django.contrib.gis`` module,
+       you need to install the GDAL library and/or enable the extension or give superuser privileges. See the requirements in the Dockerfile.
 #. Create a superuser to access the management interface: ``src/manage.py createsuperuser``
 #. To set environment variables settings, vreate a ``.env`` file. You can use and modify the provided example:
 
   .. code:: bash
 
     cp dotenv.example .env
 
+#. Generate the translation files:
+ .. code:: bash
+
+    src/manage.py makemessages -a
```

Comment:
In the backend/ dir, you should run `./bin/make_translations` (https://github.com/maykinmedia/open-archiefbeheer/blob/main/backend/bin/make_translations.sh)